shared/src/question-generators/asymptotics/between.ts

```diff
@@ -179,11 +179,10 @@ ${t("feedback.expected")}: $${variable}$.`,
       }
 
       if (variant === "nifty") {
+        // create a solution for Theta(1)
+        const solution = new ProductTerm()
```

Comment:
This should explicitly construct a constant term. Currently it depends on the default values of the `ProductTerm` constructor.